LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2016] The Kubernetes Authors All rights reserved.
```

Comment:
Two things: 
* Thing 1
```
To apply the Apache License to your work, attach the following
boilerplate notice, with the fields enclosed by brackets "[]"
replaced with your own identifying information. (Don't include
the brackets!)
```

* Thing 2: For consistency with other files, backdate to the addition of the file in the repo (2014), not today. (We don't sweepingly update copyright every year, and we've held a copyright and a license since 2014.)